sensor/common/detector/detector.go

```diff
@@ -323,7 +323,10 @@ func (d *detectorImpl) processNetworkBaselineSync(sync *central.NetworkBaselineS
 			errs.AddError(err)
 		}
 	}
-	return errs.ToError()
+	if err := errs.ToError(); err != nil {
+		return errors.Wrap(err, "failed to process network baseline sync")
+	}
+	return nil
```

Comment:
I think `errors.Wrap` is nil-aware so the branch seems unnecessary.

Reply:
I see this applies in other places of this PR too...

Reply:
I tried to be consistent with other places in our codebase.
Also I was explicitly asked once on a PR to do it like this but I cannot find this comment :(

Reply:
Found it!
- https://github.com/stackrox/rox/pull/8385#discussion_r649564253